README.md

```diff
@@ -104,13 +104,13 @@ This should make the approach generally fast and small. Actual science and optim
 Anything coercible to strings are supported:
 
 ```javascript
-const render = () => html`foo is ${foo}`;
+const render = foo => html`foo is ${foo}`;
 ```
 
 ### Attribute-value Expressions
 
 ```javascript
-const render = () => html`<div class="${blue}"></div>`;
```

Comment:
I'm not sure the changes to add parameters are necessary... The expressions are just JS that can close over variables in outer scopes. These snippets aren't mean to be complete.